lmnet/lmnet/networks/object_detection/yolo_v2.py

```diff
@@ -102,9 +102,9 @@ def __init__(
                 tf.constant(self.image_size[0], dtype=tf.int32), tf.constant(self.image_size[1], dtype=tf.int32)
             ])
 
-            # TODO(wakisaka): Be enable to cnahge `32`. it depends on pooling times.
+            # TODO(wakisaka): Be enable to change `32`. it depends on pooling times.
             # Number of cell is the spatial dimension of the final convolutional features.
-            self.num_cell = tf.tuple([tf.to_int32(self.image_size[0] / 32), tf.to_int32(self.image_size[1] / 32)])
+            self.num_cell = tf.tuple([tf.cast((self.image_size[0] / 32), tf.int32), tf.cast((self.image_size[1] / 32), tf.int32)])
```

Comment:
pep8 raise error:
```
[2019-08-16T10:04:23Z] ./lmnet/networks/object_detection/yolo_v2.py:107:121: E501 line too long (130 > 120 characters)
```
https://buildkite.com/blueoil/lmnet-test/builds/139#e7578d71-3654-4c21-8ba6-b024d45828b8

Reply:
Thanks for pointed that out, I fixed the error!